src/main/scala/com/rbmhtechnology/calliope/EventRecord.scala

```diff
@@ -24,11 +24,13 @@ case class EventRecord[A](payload: A, sourceId: String, sequenceNr: Long, creati
 
 object EventRecord {
 
-  implicit def sequenced[A]: Sequenced[EventRecord[A]] =
-    (event: EventRecord[A]) => event.sequenceNr
+  implicit def sequenced[A]: Sequenced[EventRecord[A]] = new Sequenced[EventRecord[A]] {
```

Comment:
I don't think this can be made a `val` because of its type parameter `A` which allows implicit resolution of `Sequenced[EventRecord[A]]` for arbitrary types.
For it to be a `val` the type parameter would have to be concrete.